### DISCUSSION TITLE
使用原版alist想套娃挂小雅但不成功，这是为什么

### DISCUSSION THREAD

**Discussion by witchsman** [monlor/docker-xiaoya#115] [Q&A]
我的是6.23系统的黑群晖，在docker里部署了您的小雅和emby和元数据更新三个docker项目，  但遇到两个问题，用原版list套娃挂小雅的时候，不成功，方法是根据网上说的用三段命令在ssh连接群晖下获取刷新令牌不成功，提示：Error: No such container: xiaoya  ，  获取令牌的命令没问题，在其他服务器里可以正常使用和获取。 
而且在ssh下用docker restart xiaoya 命令来重启小雅容器也不成功，也是出现上面提示。  请问这两个问题如何处理

**Comment by dierwang101** [comment]
容器名称不对，使用'docker ps --format '{{.Names}}'查看docker中的名称，再使用三行指令。
